src/Console/Commands/EncryptModel.php

```diff
@@ -32,15 +32,15 @@ public function handle()
         $class = $this->argument('model');
         $model = new $class();
 
-        $total = $model->withoutGlobalScopes()->count();
+        $total = $model->withoutGlobalScopes()->whereDoesntHaveKeystore()->count();
 
         $this->info("Found {$total} records to encrypt");
 
         $skip = 0;
         $chunk = $this->option('chunk');
 
         while ($skip < $total) {
-            $records = $model->newQuery()->withoutGlobalScopes()->skip($skip)->take($chunk)->get();
+            $records = $model->newQuery()->withoutGlobalScopes()->whereDoesntHaveKeystore()->skip($skip)->take($chunk)->get();
```

Comment:
Shouldn't this `skip` logic change? Theoretically we wouldn't need to skip any rows with this change, right?

Reply:
We should still skip to chunk records, we are batching this into smaller sets of records, so this would setup batches  to encrypt via multiple quueed jobs